californium-core/src/test/java/org/eclipse/californium/core/test/lockstep/LockstepEndpoint.java

```diff
@@ -1422,4 +1624,10 @@ public void go() {
 		 */
 		public void go() throws Exception;
 	}
+
+	public static interface MIDExpectation {
+
+		public boolean expectMID(Message message);
```

Comment:
so, if `expectMID` is to be seen in the context of these other existing methods, I would say that we probably should keep the name `expectMID` in order to not confuse people too much.
What about the interface name itself? I assume that user code does not work directly with it, right? Thus, I'd propose to use correct camel case for the interface at least ...